range.go

```diff
@@ -149,6 +150,11 @@ func (p *Plugin) EvaluateThreshold(perfData ...PerformanceData) error {
 		if perfData[i].Crit != "" {
 
 			CriticalThresholdObject := ParseRangeString(perfData[i].Crit)
+			if CriticalThresholdObject == nil {
+				err := fmt.Errorf("invalid critical threshold: %s ", perfData[i].Crit)
```

Comment:
Minor: We could adjust this in a follow-up PR, but wanted to note it while it was on my mind.

One of the approaches used thus far in this library is providing wrapped static errors. I don't know that is the *right* approach to use, but for consistency what do you think about adding to the sentinel error collection here:

https://github.com/atc0005/go-nagios/blob/7a6e9212a75d7cf063a1d0c787c74a4cb847f1a1/nagios.go#L72-L74

Something like:

```go
	// ErrInvalidRangeThreshold indicates that a given range threshold is not
	// in a supported format.
	ErrInvalidRangeThreshold = errors.New("invalid range threshold")
```

and line 154 becomes something like one of:

```go
err := fmt.Errorf("failed to parse critical range string %s: %w ", perfData[i].Crit, ErrInvalidRangeThreshold )
```

```go
err := fmt.Errorf("error parsing critical range string %s: %w ", perfData[i].Crit, ErrInvalidRangeThreshold )
```